packages/client/hmi-client/src/components/model/petrinet/tera-model-description.vue

```diff
@@ -531,11 +577,18 @@ const otherConcepts = computed(() => {
 		...(states.value?.map((s) => s.id) ?? []),
 		...(transitions.value?.map((t) => t.id) ?? [])
 	];
-	const key = Object.keys(extractions.value).find((k) => !ids.includes(k));
-	if (key) {
-		return extractions.value[key.toString()].filter((e) => e.type === 'anchored_extraction');
-	}
-	return [];
+
+	// find keys that are not aligned
+	const unalignedKeys = Object.keys(extractions.value).filter((k) => !ids.includes(k));
+
+	let unalignedExtractions: Dictionary<any>[] = [];
+	unalignedKeys.forEach((key) => {
+		unalignedExtractions = unalignedExtractions.concat(
+			extractions.value[key.toString()].filter((e) => e.type === 'anchored_extraction')
+		);
+	});
+
+	return unalignedExtractions ?? [];
```

Comment:
unaligned keys can be the string "null" or "undefined" so needed a way to pass the falsy check.  This will also map _all_ unaligned key instead of just the first one found.